lib/engine/step/g_18_co/track.rb

```diff
@@ -17,9 +17,7 @@ def process_lay_tile(action)
             # Add mine to corporation data
             @game.mine_add(action.entity)
 
-            @log << "#{action.entity.name} collects a mine token from
-              #{action.hex.name} for a total of
-              #{@game.format_currency(@game.mines_total(action.entity))} from mines"
+            @log << "#{action.entity.name} collects a mine token from #{action.hex.name}"
```

Comment:
I shortened the log message to avoid redundancy since the payout is shown on the corporation now